browser/remoteregistry.go

```diff
@@ -0,0 +1,18 @@
+package browser
+
+import "github.com/grafana/xk6-browser/env"
+
+type remoteRegistry struct {
+	isRemoteBrowser bool
+	wsURL           string
```

Comment:
One question.
We currently support the `K6_BROWSER_WS_URL` to be a comma separated list of WS URLS. In that case, the `env.IsRemoteBrowser` method returns one WS URL randomly from the list in each call.
Considering this, shouldn't we modify this method to return always the full list (if it is a list)? And then change the `IsRemoteBrowser` implementation from the new `remoteRegistry` to return one random entry for each call? This way we would be able to round robin across the list entries, otherwise all connections to the remote browser will be done to the same instance, which does not seem to fulfill our use case to support a WS URL list.

which, in that case, are used to round robin through them per each iteration. Shouldn't we change the method `env.IsRemoteBrowser` to return the whole list of WS URLs (in case it is a list) and store that in

Reply:
This PR intention was to only refactor what we currently have by moving it to the `rootModule`. I'll make the change in this [comment](https://github.com/grafana/xk6-browser/pull/889#discussion_r1197697833), and i think this suggestion may occur naturally though. If it doesn't then i'll take it into account in a subsequent PR for the next iteration that involves https://github.com/grafana/xk6-browser/issues/888.

Reply:
Ah, i get what you're saying now. I'll ensure to make that suggested change 👍